frontend/packages/ceph-storage-plugin/README.md

```diff
@@ -0,0 +1,16 @@
+# OCS UI Features
+
+The OCS UI requires some annotations in the OCS Operator CSV to perform various actions.
+
+Following table maps the annotation to its use case and accepted values:
+|Annotation Name|Purpose|Accepted Values |
+|------------------------------------------------|---------------------------|--------|
+| `features.ocs.openshift.io/enabled`| Activate UI Features | "external", "snapshot" |
```

Comment:
```suggestion
| `features.ocs.openshift.io/enabled`| Supported features from OCS backend | Array of supported features |
```

Reply:
You pass these to activate the console features.